screens/map/MapScreen.js

```diff
@@ -204,6 +225,7 @@ export default class MapScreen extends React.Component {
             </Subhead>
           </SearchBar>
         </NavHeaderContainer>
+        <CenterLocation callBack={() => this._findCurrentLocationAsync()} />
```

Comment:
Definitely nonessential but are there easy ways to make this transition smoother?